src/main/java/com/contentful/java/cma/model/CMASnapshot.java

```diff
@@ -22,4 +22,14 @@ public CMASnapshot() {
   public CMAResource getSnapshot() {
     return snapshot;
   }
+
+  /**
+   * Update the current snapshot.
+   *
+   * @param snapshot the value to be used.
+   */
+  public CMASnapshot setSnapshot(CMAResource snapshot) {
```

Comment:
What is the use case of setting a snapshot? They are read-only resources, I'm not sure I understand this 🤔